<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormMain</class>
 <widget class="QWidget" name="FormMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>690</width>
    <height>390</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="btnTest">
   <property name="geometry">
    <rect>
     <x>405</x>
     <y>5</y>
     <width>81</width>
     <height>46</height>
    </rect>
   </property>
   <property name="text">
    <string>Test</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>35</y>
     <width>56</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Version</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtDOSet">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>235</y>
     <width>61</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDI">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>180</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>DI</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDO">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>180</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>DO</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAI">
   <property name="geometry">
    <rect>
     <x>145</x>
     <y>180</y>
     <width>56</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>AI</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAO">
   <property name="geometry">
    <rect>
     <x>205</x>
     <y>180</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>AO</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="txtOut">
   <property name="geometry">
    <rect>
     <x>405</x>
     <y>60</y>
     <width>271</width>
     <height>306</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDOSet">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>225</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Set DO</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtVersion">
   <property name="geometry">
    <rect>
     <x>65</x>
     <y>40</y>
     <width>286</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtAOSet">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>275</y>
     <width>86</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>1, 3.5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAOSet">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>270</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Set AO</string>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>211</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QPushButton" name="btnService">
      <property name="text">
       <string>Service</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QCheckBox" name="chkConnected">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Connected</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="btnLogin">
      <property name="text">
       <string>Login</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QCheckBox" name="chkLogin">
      <property name="text">
       <string>Login</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="btnLock">
      <property name="text">
       <string>Lock</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QCheckBox" name="chkLock">
      <property name="text">
       <string>locked</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btnZeroing">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>70</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Zeroing</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnClean">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>10</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Clean</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
